docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
What about this `make.bat` file? Isn't it only Windows specific?

Reply:
It is; I nuked it in my fork :+1: